src/lib/_proxied-dot-io/get-static-generation-functions.ts

```diff
@@ -1,26 +1,23 @@
 import { getStaticGenerationFunctions as getStaticGenerationFunctionsBase } from '@hashicorp/react-docs-page/server'
```

Comment:
Double-checking, are the changes here just formatting changes?

Reply:
Yep!